tests/graph/test_graph_utils.py

```diff
@@ -76,7 +77,7 @@ def test_components_metrics(full_graph_edgelist: pd.DataFrame):
                     "upia": 50,
                     "upib": 50,
                     "umi": 1860,
-                    "reads": 2500,
+                    "reads": np.uint64(2500),
```

Comment:
I understand the `np` changes are for the anndata warnings, right?

Reply:
A few test cases failed due to different data types after updating dependencies. This is just to make sure the dataframe we compare against has the same column types.